src/lib.rs

```diff
@@ -35,3 +35,34 @@ pub use crate::config::{
 pub use crate::subscriber::{
     configure_subscriber, init_subscriber, ContextInfo, EventFormatter, Tracing, Uninstall,
 };
+
+/// Create a tracing error event, casting the error to &dyn [std::error::Error] for [layer::ErrorLayer],
+/// and adding the type name as error.kind.
+///
+/// Usage:
+/// ```
+/// use prima_tracing::report_error;
+///
+/// let error = "not a number".parse::<usize>().unwrap_err();
+/// report_error!(&error, "Parsing error!");
```

Comment:
I think that reference is not necessary here:
```suggestion
/// report_error!(error, "Parsing error!");
```